aspnetcore/blazor/fundamentals/configuration.md

```diff
@@ -122,49 +175,25 @@ Inject an <xref:Microsoft.Extensions.Configuration.IConfiguration> instance into
     <li>Type: @Configuration["wheels:brand:type"]</li>
     <li>Year: @Configuration["wheels:year"]</li>
 </ul>
+```
 
+Obtain a section of the configuration in C# code with <xref:Microsoft.Extensions.Configuration.IConfiguration.GetSection%2A?displayProperty=nameWithType>. The following example obtains the `wheels` section for the configuration in the preceding example:
+
+```razor
 @code {
     protected override void OnInitialized()
     {
         var wheelsSection = Configuration.GetSection("wheels");
-        
+
         ...
     }
 }
 ```
 
-To read other configuration files from the `wwwroot` folder into configuration, use an <xref:System.Net.Http.HttpClient> to obtain the file's content. When using this approach, the existing <xref:System.Net.Http.HttpClient> service registration can use the local client created to read the file, as the following example shows:
-
-`wwwroot/cars.json`:
-
-```json
-{
-    "size": "tiny"
-}
-```
-
-`Program.Main`:
-
-```csharp
-using Microsoft.Extensions.Configuration;
-
-...
-
-var http = new HttpClient()
```

Comment:
Good call to remove this now that we inject an HttpClient.

Reply:
Actually, I think I merely moved it. I'll pull it and use the existing client on the next commit.

Reply:
We inject, but not for `Program.Main`. I think the code is ok for this scenario. Let me know otherwise tho. :ear: